### ISSUE TITLE
cf Rollback - new command to rollback to prior a revision

### ISSUE THREAD

**Issue by danfein** [cloudfoundry/cli#1950]
**Please provide details on the following items. Failure to do so may result in deletion of your feature request.** 

**What's the user value of this feature request?**
An easier way to roll back to a previously saved revision of an application would reduce user effort or automation complexity, reduce risk of errors, and increase speed of response to events that require a rollback. 

**Who is the functionality for?**
Primarily human users, application developers / application operators.
In some user conversations there was some speculation around pipelines/automation triggering a rollback, but with some hesitance. 
If anyone has thoughts or opinions about how this might work in a reliable way with your current CI workflows we’d love to hear from you here in this issue or in the [#cli](https://cloudfoundry.slack.com/archives/C032824SM) channel in CloudFoundry Slack.

**How often will this functionality be used by the user?**
Whenever an app rollback is required (hopefully never? ;)

**Who else is affected by the change?**
This is a new CLI command, and would not break current workflows.

**Is your feature request related to a problem? Please describe.**
Application Revisions are a great feature, but the current API workflow is not as easy as it could be, and makes it harder to take advantage of revisions to rollback when it’s critical to do so quickly. Rolling back to a previous revision will take significantly less time than pushing the previous version of the application code.

**Describe the solution you'd like**
Proposed: a new CLI command to specify an app and revision to rollback to.

**Describe alternatives you've considered**
We considered adding a flag to the push or revision commands, but felt that the risk of rolling back by mistake was significant enough to merit its own explicit command to ensure that a rollback was only executed intentionally/consciously.

**Additional context**

```
$ cf rollback dora --revision 1

Warning: Rolling back to revision 1 for dora will create a new revision (3), with the same settings as revision 1.

Are you sure you want to continue? [y/N]> y

Creating revision 3 for app dora as admin in org my-org...
Creating app with these attributes...

+ name:       dora
  revision:   1
+ revision:   3

Creating app nora...
```

**Comment by danfein** [comment]
@pivotalrobbie @heyjcollins

**Comment by reid47** [comment]
We recently released [v7.2.0](https://github.com/cloudfoundry/cli/releases/tag/v7.2.0), which further improves the `rollback` command. Thanks for your help!
